cdm/src/main/java/ucar/nc2/ft2/coverage/adapter/SimpleGeometryCSBuilder.java

```diff
@@ -248,8 +291,9 @@ public String toString() {
     for (Dimension dim : CoordinateSystem.makeDomain(allAxes))
       f2.format("%s, ", dim.getShortName());
     f2.format("}%n");
-
-    return f2.toString();
```

Comment:
Could the variable name be camelCased? Good call on closing it!

Reply:
Thanks for catching that! Will change it accordingly. And yeah interestingly it was pretty leaky.